jwst/wfs_combine/wfs_combine.py

```diff
@@ -90,12 +94,27 @@ def do_all(self):
         im_2_a = self.create_aligned_2()  # Aligned image #2
 
         # Create and populate extensions for combined data
-        data_c, dq_c, err_c = self.create_combined(im_1_a, im_2_a)
-
+        # If the shift in x is negative, switch the two images and change the signs of the shifts
+        # This will make the final position the same even when we start at the second position.
+        if self.off_x >= 0 or not self.flip_dithers:
+            data_c, dq_c, err_c, diff_c = self.create_combined(im_1_a, im_2_a)
+        else:
+            log.info("Changing sign of dithers and flipping images")
+            self.off_x = -1 * self.off_x
+            self.off_y = -1 * self.off_y
+            data_c, dq_c, err_c, diff_c = self.create_combined(im_2_a, im_1_a)
+        log.info(f"Final x, y offset in pixels: {self.off_x} {self.off_y}")
+
+        self.diff = diff_c
         # Create a new model using the combined arrays...
         new_model = datamodels.ImageModel(data=data_c, dq=dq_c, err=err_c)
         new_model.update(self.input_1)
-
+        print(" Refine value {}".format(self.do_refine))
```

Comment:
need to change the print statement to `log.info()`

Reply:
removed it.